README.md

```diff
@@ -1,49 +1,40 @@
+# ![Remixer](small_logo.png) Remixer Android
+<img align="right" src="remixer_screenshot.png" width="300px">
 
 [![TravisCI Build Status](https://travis-ci.org/material-foundation/material-remixer-android.svg?branch=develop)](https://travis-ci.org/material-foundation/material-remixer-android) [![CircleCI Build Status](https://circleci.com/gh/material-foundation/material-remixer-android.svg?style=svg)](https://circleci.com/gh/material-foundation/material-remixer-android) [![codecov](https://codecov.io/gh/material-foundation/material-remixer-android/branch/develop/graph/badge.svg)](https://codecov.io/gh/material-foundation/material-remixer-android)
 
-# Remixer Android
+Remixer is a framework to iterate quickly on UI changes by allowing you to adjust UI variables without needing to rebuild (or even restart) your app. You can adjust Numbers, Colors, Booleans, and Strings; check out the [example app](https://github.com/material-foundation/material-remixer-android/tree/develop/remixer_example)
 
-Remixer helps teams use and refine design specs by providing an abstraction for these values that is accessible and configurable from both inside and outside the app itself.
+If you are interested in using Remixer in another platform, you may want to check out the [iOS](https://github.com/material-foundation/material-remixer-ios) and [Javascript](https://github.com/material-foundation/material-remixer-js) repos. With any of the three platforms you can use the [Remote Controller](https://github.com/material-foundation/material-remixer-remote-web).
 
-**New to Remixer?** Visit our [main repo](https://github.com/material-foundation/material-remixer) to get a full description of what it is and how it works. You can also build the Demo App and play with it. FABs are used to trigger remixer.
+## Using Remixer in your app
 
-- - -
+The following docs explain how to get Remixer up and running in your app, and details on what Variable types are currently supported.
 
-1. [Getting started](GETTING_STARTED.md)
-2. [Configure the UI](CONFIGURE_UI.md)
-3. [Project structure](PROJECT_STRUCTURE.md)
-4. [Supported Data Types](SUPPORTED_DATA_TYPES.md)
-5. [Extending Remixer](EXTENDING_REMIXER.md)
-6. API Reference (_notice this includes internal APIs_):
-  - [Current Version 0.6.6](https://jitpack.io/com/github/material-foundation/material-remixer-android/remixer/0.6.6/javadoc/index.html)
-  - [Develop Snapshot](https://jitpack.io/com/github/material-foundation/material-remixer-android/remixer/develop-SNAPSHOT/javadoc/index.html)
+- [Using Remixer](USING_REMIXER.md)
+  - [Configure the UI](CONFIGURE_UI.md)
+  - [Supported Data Types](SUPPORTED_DATA_TYPES.md)
+  - (Optionally) [Configure the Firebase Remote Controller](CONFIGURE_FIREBASE.md)
 
-- - -
+## Contributing to Remixer
 
-## State of development
+We're excited you want to contribute to the project! Please read these docs if you're interested so we can get your contributions submitted quickly.
```

Comment:
These two sentences are a little contradictory :) I'd remove the "if you're interested" from the second one as the first one assumes you are.